src/components/TriggerSignIn.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { useRownd } from '../context';
+import { SignInProps } from '../context/types';
+
+interface TriggerSignInProps {
+  children: React.ReactNode;
+  components: {
+    Initializing?: React.ReactNode;
+    Unauthenticated?: React.ReactNode;
+  };
+  signInProps?: SignInProps;
+}
+
+const TriggerSignIn: React.FC<TriggerSignInProps> = ({
```

Comment:
Thoughts on a name like `WithAuthentication` or `Authenticated`?

Reply:
It's been updated to `RequireSignIn`. 

We discussed `RequreAuthentication` but I thought it would be better to match the `requestSignIn()` hook.